src/events/processTransfers/index.js

```diff
@@ -29,10 +29,15 @@ function processTransfersBuilder(config) {
       validatorContract = new web3Home.eth.Contract(bridgeValidatorsABI, validatorContractAddress)
```

Comment:
Please log the received value of `validatorContractAddress`.

Reply:
Done in fbfcd76a613071a26c1d722bac226ffeda8853a2.